### PR TITLE
Lunamedia ad size parameter update

### PR DESCRIPTION
**Pull request by trchandraprakash**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by trchandraprakash** [comment]
Hi Guys, Can you please let me know if all good here and when i can make it available for clients. Thanks

File: modules/lunamediaBidAdapter.js [hunk 1]
```diff
@@ -210,8 +211,18 @@ function createVideoRequestData(bid, bidderRequest) {
...
+
+  var firstSize = [];
+  var sizes = [];
+
+  if (typeof paramSize !== 'undefined' && paramSize != '') {
+    sizes = parseSizes(paramSize);
+  } else {
+    sizes = getVideoSizes(bid);
+  }
+  firstSize = getFirstSize(sizes);
```
Comment (msm0504) [inline modules/lunamediaBidAdapter.js:225]: Change this to `let firstSize = getFirstSize(sizes);` and remove line 217

Comment (msm0504) [inline modules/lunamediaBidAdapter.js:225]: Change this to `const firstSize = getFirstSize(sizes);` and remove line 217

File: modules/lunamediaBidAdapter.js [hunk 2]
```diff
@@ -210,8 +211,18 @@ function createVideoRequestData(bid, bidderRequest) {
   let topLocation = getTopWindowLocation(bidderRequest);
   let topReferrer = getTopWindowReferrer();
 
-  let sizes = getVideoSizes(bid);
-  let firstSize = getFirstSize(sizes);
+  // if size is explicitly given via adapter params
+  var paramSize = getVideoBidParam(bid, 'size');
```
Comment (msm0504) [inline modules/lunamediaBidAdapter.js:215]: Why is `var` being used in the code in this PR when `let` and `const` are used throughout the existing code?

File: modules/lunamediaBidAdapter.md [hunk 3]
```diff
@@ -29,7 +29,8 @@ var displayAdUnit = [
         bidder: 'lunamedia',
         params: {
             pubid: '121ab139faf7ac67428a23f1d0a9a71b',
-            placement: 1234
+			placement: 1234,
+			size="320x50"
```
Comment (msm0504) [inline modules/lunamediaBidAdapter.md:33]: Change `=` to `:`

Same change on line 57

File: modules/lunamediaBidAdapter.js (hunk 1 above)
Comment (trchandraprakash) [reply modules/lunamediaBidAdapter.js]: Hi @msm0504 updated as requested.

File: modules/lunamediaBidAdapter.js (hunk 2 above)
Comment (trchandraprakash) [reply modules/lunamediaBidAdapter.js]: updated code.

File: modules/lunamediaBidAdapter.md (hunk 3 above)
Comment (trchandraprakash) [reply modules/lunamediaBidAdapter.md]: changes are done.

**Pull request merged**